packages/core/admin/ee/server/validation/audit-logs.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const { yup, validateYupSchema } = require('@strapi/utils');
+
+const ALLOWED_SORT_STRINGS = ['action:ASC', 'action:DESC', 'createdAt:ASC', 'createdAt:DESC'];
```

Comment:
`createdAt` is specific to the local provider, what we should use instead is `date`